README.md

```diff
@@ -37,10 +37,10 @@ env::var("DEP_GDK_CFG").unwrap() == "gdk_3_4 gdk_3_6 gdk_3_8 gdk_3_10 gdk_3_12"
 ```
 
 will build the [generator](https://github.com/gkoz/gir) in `gir` and
-generate the crates using specifications from `conf` and [GIR definitions]
-(https://github.com/gkoz/gir-files) from `gir-files`.
+generate the crates using specifications from `conf` and
+[GIR definitions](https://github.com/gkoz/gir-files) from `gir-files`.
 
-Currently `gir-files` is populated from the [gnome-staging ppa]
-(https://launchpad.net/~gnome3-team/+archive/ubuntu/gnome3-staging).
+Currently `gir-files` is populated from the
+[gnome-staging ppa](https://launchpad.net/~gnome3-team/+archive/ubuntu/gnome3-staging).
```

Comment:
Currently we use http://packages.ubuntu.com/zesty/amd64/ for most of packages

Reply:
Update this one too?

Reply:
Hm, I was wrong it from "artful" now.
Seems text need be "polulated from Ubuntu packages"